src/code42cli/cmds/securitydata.py

```diff
@@ -149,7 +160,7 @@ def file_event_options(f):
     f = process_owner_option(f)
     f = tab_url_option(f)
     f = include_non_exposure_option(f)
-    f = extraction_format_option(f)
+    f = file_events_format_option(f)
```

Comment:
Alerts uses the standard format option (doesn't support CEF), so that is why this is renamed